src/window.js

```diff
@@ -68,6 +79,33 @@ var ToolboxWindow = GObject.registerClass({
         });
     }
 
+    _startMusic() {
```

Comment:
Instead of implementing _startMusic and _stopMusic, could you use the SoundItem class from https://github.com/endlessm/gnome-shell/blob/master/js/misc/soundServer.js#L34-L83 ?